contracts/contracts/strategies/ConvexEthMetaStrategy.sol

```diff
@@ -310,7 +542,7 @@ contract ConvexEthMetaStrategy is InitializableAbstractStrategy {
     }
```

Comment:
In `checkBalance` the initial `balance += address(this).balance;` doesn't need to be `+=`, but could just be `=`?

Reply:
yes, and we save 70 gas

Before
```
|  Contract               ·  Method                              ·  Min        ·  Max        ·  Avg        ·  # calls      ·  eur (avg)  │
··························|······································|·············|·············|·············|···············|··············
|  MockVault              ·  checkBalance                        ·          -  ·          -  ·      59858  ·            1  ·          -  │
```
After
```
|  Contract               ·  Method                              ·  Min        ·  Max        ·  Avg        ·  # calls      ·  eur (avg)  │
··························|······································|·············|·············|·············|···············|··············
|  MockVault              ·  checkBalance                        ·          -  ·          -  ·      59788  ·            1  ·          -  │
```